### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by rocky**
Although this is a good thing to do in of itself. It will also be useful should we want to join NumFocus (which I am in favor of).

### CODE REVIEW

**Comment by mmatera** [comment]
LGTM

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+like social media. Violating these terms may lead to a temporary or
+permanent ban.
+
+### 3. Temporary Ban
+
+**Community Impact**: A serious violation of community standards, including
+sustained inappropriate behavior.
+
+**Consequence**: A temporary ban from any sort of interaction or public
+communication with the community for a specified period of time. No public or
```
Comment (TiagoCavalcante) [inline CODE_OF_CONDUCT.md:101]: Should we specify this?

**Review by TiagoCavalcante — APPROVED** [review]
LGTM

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (rocky) [reply CODE_OF_CONDUCT.md:101]: I selected the "Contributor Covenant" text unmodified. Personally, I don't want to get into the business of customizing this.

It's sort of how you don't like it when there isn't a newline after triple quotes in a docstring. 

But joking aside, if we start to customize this then it makes it harder for organizations like NumFocus to decide whether they want to do "business" with us.

Comment (mmatera) [reply CODE_OF_CONDUCT.md:101]: I think that since we do not have any experience with this kind of behavior issue, the standard is OK.

**Comment by rocky** [comment]
@TiagoCavalcante would you do the same for mathics-threejs-backend and Django? 

I'll do the other repositories when I get a chance or anyone feel free to just do the same for the others. Thanks.

**Comment by TiagoCavalcante** [comment]
@rocky yes, I can do that.

**Pull request merged**